src/ImageProcessorCore/Formats/Tiff/ValueDecoders/TiffUShortValueDecoder.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace ImageProcessorCore.Formats.Tiff.ValueDecoders
+{
+    public class TiffUShortValueDecoder : ITiffValueDecoder
+    {
+        public bool DecodeValue(TiffReader reader, IFDEntry entry, TiffTagValue value)
+        {
+            if (entry.FieldType != IFDEntryType.UShort)
```

Comment:
Just a wee style note here, I have full blown StyleCop :cop: OCD so I'd really like to use braces, avoid `var` etc as much as possible please.